proposals/cloudedgeclientCG/charter.md

```diff
@@ -13,7 +13,7 @@ The mission of this group is to provide mechanisms and interfaces between Centra
 ## Scope of Work
 This group aims to discuss proposals for Client-Edge-Cloud coordination, including:
 * Use cases and requirements for Client-Edge-Cloud coordination
-* Proposals for APIs and mechanisms that enable computing workload offloading and orchestration between Central Cloud, Edge Cloud, and Client
+* Proposal, design, and incubation of APIs and mechanisms that enable computing workload offloading and orchestration between Central Cloud, Edge Cloud, and Client.
```

Comment:
Yes, I think API incubation is not outside of scope for CGs in general.

Reply:
I think there's still something to clarify, as the current draft (and https://github.com/w3c/web-networks/pull/25) says the group won't produce specifications (and it certainly can't produce Recommendation track specs), but does the group intend to work on API designs, or stop at use cases and requirements?

Reply:
+1 - I think the group should include producing specifications in its scopes (even though they would obviously not be standards-track at that stage yet); L34 forbids it at the moment, which I don't think it should

Reply:
> does the group intend to work on API designs 

The group needs to decide that. There are several APIs that have been incubated/developed in a CG (whether it continued or not on the standardization track) - but keep in mind the difference between CG vs. WG, and also these notes:
https://www.w3.org/standards/types/#x2-pre-standardization-proposals-notes
https://www.w3.org/policies/process/ 
When an API reaches a certain maturity, it makes sense to continue developing it in a WG on the standardization track.
But the proposed formulation for incubating API development is fine for a CG.